docs/user/hbase/install.rst

```diff
@@ -105,47 +105,52 @@ registered manually. See :ref:`coprocessor_alternate` for details.
-    installation or distribution of Hadoop you are using and may not be reliable.
+GeoMesa comes with a set of command line tools for managing HBase features located in
+``geomesa-hbase_2.11-$VERSION/bin/`` of the binary distribution.
 
-    3. If no environmental variables are set but the ``hbase`` and ``hadoop`` commands are available then GeoMesa will
-    interrogate them for their classpaths by running the ``hadoop classpath`` and ``hbase classpath`` commands. This
-    method of classpath determination is slow due to the fact that the ``hbase classpath`` command forks a new JVM. It
-    is therefore recommended that you set manually set these variables in your environment or the
-    ``conf/geomesa-env.sh`` file.
+GeoMesa requires ``java`` to be available on the default path.
 
-In addition, ``geomesa-hbase`` will pull any additional entries from the ``GEOMESA_EXTRA_CLASSPATHS``
-environment variable.
+Configuring the Classpath
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+GeoMesa needs HBase and Hadoop JARs on the classpath. These are not bundled by default, as they should match
+the versions installed on the target system.
 
-Note that the ``GEOMESA_EXTRA_CLASSPATHS``, ``GEOMESA_HADOOP_CLASSPATH``, and ``GEOMESA_HBASE_CLASSPATH`` variables
-all follow standard
-`Java Classpath <http://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html>`_ conventions, which
-generally means that entries must be directories, JAR, or zip files. Individual XML files will be ignored. For example,
-to add a ``hbase-site.xml`` or ``core-site.xml`` file to the classpath you must either include a directory on the
-classpath or add the file to a zip or JAR archive to be included on the classpath.
+If the environment variables ``$HBASE_HOME`` and ``$HADOOP_HOME`` are set, then GeoMesa will load the appropriate
+JARs and configuration files from those locations and no further configuration is required. For simplicity,
+environment variables can be specified in ``geomesa-hbase_2.11-$VERSION/conf/geomesa-env.sh``.
 
-Use the ``geomesa classpath`` command in order to see what JARs are being used.
+For advanced scenarios, the environmental variables ``GEOMESA_HADOOP_CLASSPATH`` and ``GEOMESA_HBASE_CLASSPATH``
+can be set to override all other logic.
 
-A few suggested configurations are below:
+If no environment variables are set but the ``hbase`` and ``hadoop`` commands are available, then
+GeoMesa will interrogate them for their classpaths by running the ``hadoop classpath`` and ``hbase classpath``
+commands. Note that this can be slow, so it is usually better to use ``GEOMESA_HADOOP_CLASSPATH`` and
+``GEOMESA_HBASE_CLASSPATH`` as described above.
 
 .. tabs::
 
+    .. group-tab:: Standard
+
+        Configure GeoMesa to use pre-installed HBase and Hadoop distributions:
+
+        .. code-block:: bash
+
+            export HADOOP_HOME=/path/to/hadoop
+            export HBASE_HOME=/path/to/hbase
```

Comment:
I always find HADOOP_HOME, and HBASE_HOME somewhat frustrating on a managed cluster. We think of these paths as "the place where the tarball was extracted," or the path to the hadoop executable, but what do we actually need from them? I'm looking at an executable at /usr/bin/hadoop, and a core-site.xml in  /etc/hadoop/conf/core-site.xml. How do I know if these will work or if I need to use more advanced variables?

Reply:
there's a separate tab for EMR here, does that suit your use case? https://github.com/locationtech/geomesa/pull/2509/files/57736e5315c87316e2a4655306080fbe27bfa70f#diff-9afc2c38411542d4ba4bb4656a2b3500R154
I'm not sure if there's a good way to know when you need different configuration setups... the most control is through using the explicit classpaths as described here:
https://github.com/locationtech/geomesa/pull/2509/files/57736e5315c87316e2a4655306080fbe27bfa70f#diff-9afc2c38411542d4ba4bb4656a2b3500R136
we could probably update our scripts to be better about pulling out hadoop/hbase paths from standard managed install locations... I think generally we look for the 'conf' folder and any jar files and add them all to the classpath.